admin/jobs/river/org_jobs.go

```diff
@@ -173,3 +173,77 @@ func (w *DeleteOrgWorker) Work(ctx context.Context, job *river.Job[DeleteOrgArgs
 
 	return nil
 }
+
+type LogInactiveOrgsArgs struct{}
+
+func (LogInactiveOrgsArgs) Kind() string { return "log_inactive_orgs" }
+
+type LogInactiveOrgsWorker struct {
+	river.WorkerDefaults[LogInactiveOrgsArgs]
+	admin  *admin.Service
+	logger *zap.Logger
+}
+
+func (w *LogInactiveOrgsWorker) Work(ctx context.Context, job *river.Job[LogInactiveOrgsArgs]) error {
+	orgs, err := w.admin.DB.FindInactiveOrganizations(ctx)
+	if err != nil {
+		if errors.Is(err, database.ErrNotFound) {
+			return nil
+		}
+		return fmt.Errorf("failed to find inactive organizations: %w", err)
+	}
+
+	for _, org := range orgs {
+		projects, err := w.admin.DB.FindProjectsForOrganization(ctx, org.ID, "", 100)
+		if err != nil {
+			return fmt.Errorf("failed to find projects for organization %s: %w", org.Name, err)
+		}
+		for _, proj := range projects {
+			p, err := w.admin.HibernateProject(ctx, proj)
+			if err != nil {
+				return fmt.Errorf("failed to hibernate project %s: %w", proj.ID, err)
+			}
+			w.logger.Warn("hibernated project", zap.String("project_id", p.ID), zap.String("project_name", p.Name), zap.String("org_id", org.ID), zap.String("org_name", org.Name))
+		}
+		w.logger.Warn("inactive organization", zap.String("org_id", org.ID), zap.String("org_name", org.Name), zap.Time("last_updated_at", org.UpdatedOn), zap.Int("connected_projects", len(projects)))
+	}
+
+	return nil
+}
+
+type DeleteInactiveOrgsArgs struct {
+	OrgID string
+}
+
+func (DeleteInactiveOrgsArgs) Kind() string { return "delete_inactive_orgs" }
+
+type DeleteInactiveOrgsWorker struct {
```

Comment:
1. Since this takes an `OrgID`, shouldn't the job have a singular name, i.e. `DeleteInactiveOrg`?
2. Strictly speaking this job works for any org, there is nothing in it that has "inactive" handling

In light of the above, shouldn't this logic just be merged into the `DeleteOrg` job? I.e. adapt that job to check if the org has any projects, and then delete those (seems like an oversight in that job anyway).